tapalcatl_server/handler.go

```diff
@@ -116,6 +121,10 @@ func logBool(x bool) string {
 	}
 }
 
+func logDuration(x time.Duration) string {
+	return fmt.Sprintf("%d", x.Nanoseconds())
```

Comment:
Even in the statsd metrics? ie remove the guard here? https://github.com/tilezen/tapalcatl/pull/14/files#diff-c90595d9383262ef8aa9cba693f6ced6R246